modules/tripleliftBidAdapter.js

```diff
@@ -364,4 +377,89 @@ function _buildResponseObject(bidderRequest, bid) {
   return bidResponse;
 }
 
+function getSafeFrameApi() {
+  const safeFrameApi = _getApi(SAFE_FRAME_API_NAME);
+  if (safeFrameApi && safeFrameApi.ext) {
```

Comment:
I don't get ut, the creative returned from gam might still be a safeframe? #4796 is related